inf-clojure.el

```diff
@@ -392,7 +392,7 @@ This should usually be a combination of `inf-clojure-prompt' and
   :package-version '(inf-clojure . "2.0.0"))
 
 (defvar inf-clojure-buffer nil
-  "The current inf-clojure process buffer.
+  "The current ‘inf-clojure’ process buffer.
```

Comment:
It should be `\`inf-clojure'`.

Reply:
Yeah, I just think you didn't put the right quotes around it. Only the first one should be a backquote.